sdk_container/src/third_party/coreos-overlay/dev-lang/rust/rust-1.70.0.ebuild

```diff
@@ -3,6 +3,7 @@
 
 EAPI=8
 
+# Flatcar: skip python3_12 which is not supported
```

Comment:
Was this change necessary? Python 12 is masked, so we will use python 3.10 anyway. I mean - the change itself is not a problem, just checking if we can minimize the diff with Gentoo.

Reply:
It was necessary.
I have just tested again with adding back python3_12. Then it immediately failed.

```
Calculating dependencies / * ERROR: dev-lang/rust-1.70.0::coreos failed (depend phase):
 *   Invalid implementation in PYTHON_COMPAT: python3_12
```

Reply:
Interesting, probably a recent update in python eclass is needed. Anyway, this will come with weekly updates, so let's keep this modification for now.